.github/workflows/test-changed.yml

```diff
@@ -1,6 +1,10 @@
-name: Push/PR
+name: Test Modified Packages
 
-on: [push, pull_request]
+on: 
+  pull_request:
+    branches:
+      - '*'
+      - '!master'
```

Comment:
What does excluding master for PR mean? Don't we always create PR against master?

Reply:
Yes, I think I was confused about what "branches" means for PR. Fixing.